core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -7791,7 +7791,7 @@ void CmpSeabaseDDL::alterSeabaseTableAddPKeyConstraint(
   // this new primary key.
   // Do this optimization in mode_special_4 only.
   Lng32 len = 0;
-  Int64 rowCount = 0;
+  Lng32 rowCount = 0;
```

Comment:
Checked again and even though the variable is named rowcount, we are retrieving the value "[ANY 1]" into that. So this does return a 4 byte interger and we're ok.  Only the (count*) return values are 8 bytes long.